developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md

```diff
@@ -9,95 +9,107 @@ ms.topic: "article"
-                jobDefParameters.Add("ScriptBlock", scriptBlock);  // A scriptblock or script FilePath
-                                                                   // is required.
+  ScriptBlock scriptBlock = ScriptBlock.Create(@"1..5 | foreach {sleep 1; ""SchedJobOutput $_""}");
+  jobDefParameters.Add("ScriptBlock", scriptBlock);  // A scriptblock or script FilePath
+                                                    // is required.
 
 ```
 
-## Creating the Invocation and Job Definition Objects
+## Creating the invocation and job definition objects
 
-You then create ScheduledJobInvicationInfo and SheduledJobDefinition objects to run the job. The following code demonstrates this.
+You then create `ScheduledJobInvocationInfo` and `ScheduledJobDefinition` objects to run the job as
+shown in the following example:
 
 ```csharp
 ScheduledJobInvocationInfo jobInvocationInfo = new ScheduledJobInvocationInfo(
-                    new JobDefinition(typeof(ScheduledJobSourceAdapter), scriptBlock.ToString(), schedJobDefName),
-                    jobDefParameters);
+    new JobDefinition(typeof(ScheduledJobSourceAdapter), scriptBlock.ToString(), schedJobDefName),
+    jobDefParameters);
 
-                schedJobDefinition = new ScheduledJobDefinition(
-                    jobInvocationInfo,                          // Defines the PowerShell job to run.
-                    new ScheduledJobTrigger[1] { jobTrigger },  // Defines when to run the PowerShell job.
-                    null,                                       // No custom options.  We accept default.
-                    null);                                      // No credentials.  PowerShell job is run
-                                                                // in default Task Scheduler process, account.
+    schedJobDefinition = new ScheduledJobDefinition(
+    jobInvocationInfo,                          // Defines the PowerShell job to run.
+    new ScheduledJobTrigger[1] { jobTrigger },  // Defines when to run the PowerShell job.
+    null,                                       // No custom options. We accept default.
+    null);                                      // No credentials. PowerShell job is run
+                                                // in default Task Scheduler process, account.
 
 ```
 
-## Registering the Job with the Task Scheduler
+## Registering the job with the task scheduler
 
 The following code registers the job with the [Windows Task Scheduler](http://go.microsoft.com/fwlink/?LinkId=251817).
 
 ```csharp
 schedJobDefinition.Register();
-                registrationSucceeded = true;
-                Console.WriteLine("Scheduled job has been registered.  Waiting 30 seconds for it to be started and run.");
+    registrationSucceeded = true;
+    Console.WriteLine("Scheduled job is registered. Waiting 30 seconds for it to run.");
 
 ```
 
-## Complete Code Example
+## Complete code Example
```

Comment:
To stay within the line character limits, I inserted some line breaks in the code sample.

Reply:
You didn't need to break those lines. I fixed it.

Reply:
Thank you Sean!